desafio-02/edipocba/python/desafio-02.py

```diff
@@ -0,0 +1,8 @@
+"""Este código imprime os números primos entre 1 e 10000."""
+for numero in range(2, 10001):
+    contador = 0
+    for numeroDivisao  in range(1, numero+1):
```

Comment:
Não faz sentido estar com um número maior do que número (i.e. numero+1) pois nenhum número é divisivel por um número maior do que ele.

Reply:
Na linguagem de programação python o código "for n in range(1, 11)" executa um loop do número 1 até o 10, ou seja, ele não considera o último número, por isso eu coloquei 1 número maior

Reply:
Coloque então in range (2, numero) pois assim só vai testar até número-1 pois você sabe que todo número é divisivel por si mesmo.

Reply:
Notei aqui que você já implementou a modificação sugerida.